packages/ERTP/src/amountMath.js

```diff
@@ -194,47 +220,124 @@ const AmountMath = {
     // Will throw on inappropriate value
     return AmountMath.make(brand, allegedValue);
   },
+  /**
+   * Extract and return the value.
+   *
+   * @template {AmountValue} [V=AmountValue]
+   * @param {Brand} brand
+   * @param {Amount<V>} amount
+   * @returns {V}
+   */
   getValue: (brand, amount) => AmountMath.coerce(brand, amount).value,
+  /**
+   * Return the amount representing an empty amount. This is the
+   * identity element for MathHelpers.add and MatHelpers.subtract.
+   *
+   * @param {Brand} brand
+   * @param {AssetKind=} assetKind
+   * @returns {Amount}
+   */
   makeEmpty: (brand, assetKind = AssetKind.NAT) => {
     assertRemotable(brand, 'brand');
     assertAssetKind(assetKind);
     const value = helpers[assetKind].doMakeEmpty();
     return harden({ brand, value });
   },
+  /**
+   * Return the amount representing an empty amount, using another
+   * amount as the template for the brand and assetKind.
+   *
+   * @template {AmountValue} [V=AmountValue]
+   * @param {Amount<V>} amount
+   * @returns {Amount<V>}
+   */
   makeEmptyFromAmount: amount => {
     assertRecord(amount, 'amount');
     const { brand, value } = amount;
     const assetKind = assertValueGetAssetKind(value);
+    // @ts-ignore assetKind not sufficiently typed but it's guaranteed to match V
```

Comment:
Why not `@ts-expect-error`?

Reply:
Due to the `tsc` command finding this problem but not my IDE. (IDE lists a problem if error expected here) I've got a backburner project to fix that https://github.com/Agoric/agoric-sdk/pull/4404